File: The Core/At The Crossroads/13. Arithmetic Expression/arithmeticExpression.py
def arithmeticExpression(a, b, c):

    divInt = False
    plus = a + b
    sub = a - b
    mult = a * b
    
    if a % b == 0:
        divInt = True
    else: 
        divInt = False
        
    if plus == c or sub == c or mult == c:
        return True
    elif divInt == True:
        if a / b == c:
            return True
        else:
            return False
        
    return False

File: The Core/At The Crossroads/13. Arithmetic Expression/arithmeticExpression.lua
function arithmeticExpression(a, b, c)
    
    divInt = false
    plus = a + b
    sub = a - b
    mult = a * b
    
    if a % b == 0 then
        divInt = true
    else 
        divInt = false
    end
    
    if plus == c or sub == c or mult == c then
        return true
    elseif divInt == true then
        if a / b == c then
            return true
        else
            return false
        end
    else
        return false
    end
end
